.travis.yml

```diff
@@ -42,12 +47,46 @@ matrix:
     - name: "LLVM 8 Release"
       env: LLVM_VERSION=8 BASE=bionic TYPE=Release RUN_ALL_TESTS=1 RUNTIME_TEST_DISABLE=builtin.cgroup
 
+    - name: "Static+glibc-2.27 Debian-linked LLVM 8 Debug"
+      env: LLVM_VERSION=8 BASE=bionic-static STATIC_LINKING=ON STATIC_LIBC=OFF EMBED_LLVM=OFF EMBED_CLANG=ON EMBED_LIBCLANG_ONLY=ON TYPE=Debug RUN_ALL_TESTS=1 RUNTIME_TEST_DISABLE=builtin.cgroup
+    - name: "Static+glibc-2.27 Debian-linked LLVM 8 Release"
+      env: LLVM_VERSION=8 BASE=bionic-static STATIC_LINKING=ON STATIC_LIBC=OFF EMBED_LLVM=OFF EMBED_CLANG=ON EMBED_LIBCLANG_ONLY=ON TYPE=Release RUN_ALL_TESTS=1 RUNTIME_TEST_DISABLE=builtin.cgroup
+
+    # This job exists to warm the build cache, it will succeed on timeout so that
+    # it can upload its progress and continue in a subsequent run. It shares a
+    # cache with the Release version of this step, which will still fail on
+    # timeout until the cache is warm. To warm the cache trigger a rebuild 1-3
+    # times, either through the UI or with an empty commit.
+    - name: "Static+glibc-2.27 Vanilla LLVM 8 - LLVM artifact cache build - Release"
```

Comment:
I couldn't get this to build successfully for Debug mode on travis, it takes too long and is too big. It generates nearly 40GB of artifacts, and timeouts even trying to load the cache, if it's able to save it.

Reply:
The substitute is to still build debug mode, but link libclang only.